crates/testsuite/features/cmd.feature

```diff
@@ -150,6 +150,6 @@ Feature: Rooch CLI integration tests
     Scenario: rpc test
       Given a server for rpc
       Then cmd: "rpc request --method eth_getBalance --params \"0x1111111111111111111111111111111111111111\""
-      Then assert: "{{$.rpc[-1]}} == 0x56bc75e2d63100000"
+      Then assert: "{{$.rpc.ethereum[-1]}}" equals "0x56bc75e2d63100000"
```

Comment:
The response must be the same as the ETH RPC response.

https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getbalance

```
// Request
curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getBalance","params":["0x407d73d8a49eeb85d32cf465507dd71d507100c1", "latest"],"id":1}'
// Result
{
  "id":1,
  "jsonrpc": "2.0",
  "result": "0x0234c8a3397aab58" // 158972490234375000
}
```

Reply:
Why the RPC response result contains an `ethereum` field?

Reply:
I don't know where to handle this, but in the original code it had an error:

>    ✘  Then assert: "{{$.rpc}} == 0x56bc75e2d63100000"
      Step failed:
      Defined: features/cmd.feature:7:7
      Matched: crates/testsuite/tests/integration.rs:102:1
      Step panicked. Captured output: assertion failed: `(left == right)`
        left: `"[Error: RPC call failed: ErrorObject { code: MethodNotFound, message: \"Method not found\", data: None }]"`,
       right: `"0x56bc75e2d63100000"`: Assert "[Error: RPC call failed: ErrorObject { code: MethodNotFound, message: \"Method not found\", data: None }]" == "0x56bc75e2d63100000" failed

Reply:
I changed it to `{{$.result}}`.